lib/checks/tables/same-caption-summary-evaluate.js

```diff
@@ -1,12 +1,27 @@
-import { accessibleText } from '../../commons/text';
-
-function sameCaptionSummaryEvaluate(node) {
-  // passing node.caption to accessibleText instead of using
-  // the logic in accessibleTextVirtual on virtualNode
-  return (
-    !!(node.summary && node.caption) &&
-    node.summary.toLowerCase() === accessibleText(node.caption).toLowerCase()
-  );
-}
+import { sanitize, subtreeText } from '../../commons/text';
 
 export default sameCaptionSummaryEvaluate;
+
+function sameCaptionSummaryEvaluate(node, options, virtualNode) {
+  if (virtualNode.props.nodeName !== 'table') {
+    return undefined;
```

Comment:
As I said yesterday (https://github.com/dequelabs/axe-core/pull/3604#discussion_r945902734) this isn't necessary. We don't need to check that the node matches the selector in the check. Axe has more than enough tests to make sure of that.

Reply:
Thank you Wilco, makes sense re: selector responsibility for this.